src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -99,5 +94,17 @@ public async Task<IActionResult> GetBranchesAsync(string resourceUri)
             object response = await _devOpsClient.GetBranchesAsync(resourceUri, this.HttpContext.Request.Headers[RequestIdHeaderName]);
             return Ok(response);
         }
+
+        [HttpGet(UriElements.DevOpsConfig)]
+        public async Task<IActionResult> GetResourceProviderConfig(string resourceProviderType)
```

Comment:
Thinking out loud here from security standpoint:
Do we need to pass the entire config as an API response to client (in this case applens).
This config might have default approvers aliases or groups in future.  Maybe we just need the default branch for the UI purpose.

From Applens standpoint, even though user needs to be authenticated but once you are authenticated(even bad actor), today you can execute any runtime host controller API via Applens Backend (and Durian is not applicable here since this is not case specific data)

Reply:
Agreed. This was mainly for checking if Resource Provider type has been enabled for graduation or not. I can make changes to just return bool `true` or `false`